addons/binding/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/handler/NetatmoModuleHandler.java

```diff
@@ -43,7 +46,17 @@ protected NetatmoModuleHandler(@NonNull Thing thing) {
     @Override
     public void initialize() {
         super.initialize();
-        requestParentRefresh();
+        refreshJob = scheduler.schedule(() -> {
+            requestParentRefresh();
+        }, 5, TimeUnit.SECONDS);
```

Comment:
Every 5 seconds feels often, why?

Reply:
No it is only one one shot task delayed of 5 seconds.

Reply:
Of course, thanks!